common/src/test/groovy/com/okta/cli/common/model/OidcPropertiesTest.groovy

```diff
@@ -63,6 +67,95 @@ class OidcPropertiesTest {
         oidcProperties.setRedirectUris(List.of("http://localhost:8080/login/oauth2/code/oidc"))
         def clientProperties3 = oidcProperties.getProperties()
         assertThat clientProperties3.get("quarkus.oidc.authentication.redirect-path"), is("/login/oauth2/code/oidc")
+    }
+
+    @Test
+    void jhipsterQuarkusServicePropertiesDetectionTest() {
+
+        String packageJson ="""
+            "devDependencies": {
+                "generator-jhipster": "6.10.5",
+                "generator-jhipster-quarkus": "1.0.0",
+            }
+        """.stripLeading()
+
+        Map<String, String> expected = [
+            "quarkus.oidc.auth-server-url": "https://issuer.example.com",
+            "quarkus.oidc.client-id": "test-client-id",
+            "quarkus.oidc.credentials.secret": "test-client-secret",
+            "quarkus.oidc.authentication.redirect-path": "/login/oauth2/code/oidc",
+            "quarkus.oidc.application-type": "service"
+        ]
+
+        jhipsterPropertiesTest(packageJson, expected, OpenIdConnectApplicationType.SERVICE)
+    }
+
+    @Test
+    void jhipsterQuarkusPropertiesDetectionTest() {
+
+        String packageJson ="""
+            "devDependencies": {
+                "generator-jhipster": "6.10.5",
+                "generator-jhipster-quarkus": "1.0.0",
+            }
+        """.stripLeading()
+
+        Map<String, String> expected = [
+                "quarkus.oidc.auth-server-url": "https://issuer.example.com",
+                "quarkus.oidc.client-id": "test-client-id",
+                "quarkus.oidc.credentials.secret": "test-client-secret",
+                "quarkus.oidc.authentication.redirect-path": "/login/oauth2/code/oidc",
+                "quarkus.oidc.application-type": "web-app"
```

Comment:
The default is `hybrid`: https://github.com/jhipster/generator-jhipster-quarkus/blob/main/generators/server/templates/quarkus/src/main/resources/application.properties.ejs#L87

Are you changing this because Okta needs it or JHipster does. @danielpetisme Do you know what the difference is between `web-app`, `service`, and `hybrid` is when it comes to JHipster?

Reply:
>Are you changing this because Okta needs it or JHipster does. @danielpetisme Do you know what the difference is between web-app, service, and hybrid is when it comes to JHipster?

I do! I wasted way too much hours on this...
- `service`:  targets system to system authentication (ie. microservices seeking for authentication, or in unit tests)
- `web-app`: targets a user dealing with a UI (like when you're actually using the JHipster web app)
- `hybrid`: well it's a mix of the 2 above
More info:
https://github.com/quarkusio/quarkus/blob/master/extensions/oidc/runtime/src/main/java/io/quarkus/oidc/OidcTenantConfig.java#L775

Reply:
In OAuth terms, it sounds like service is a resource server, a web app is for a UI that can redirect, and a hybrid is for both. JHipster's Spring Security support uses "hybrid" for its monoliths and gateways so you can log in AND they can be a resource server (for JHipster's Ionic and React Native modules). For microservices, it just sets up a resource server.

IMO, it doesn't seem like the Okta CLI should override this value. It should just default to what JHipster has.